composer.json

```diff
@@ -14,7 +17,14 @@
         }
     ],
     "autoload": {
-        "psr-4": { "Facile\\DoctrineMySQLComeBack\\Doctrine\\DBAL\\": "src/" }
+        "psr-4": { "Facile\\DoctrineMySQLComeBack\\Doctrine\\DBAL\\": "src/" },
+        "exclude-from-classmap": [
```

Comment:
Tests should not be included in the classmap, but should be better to add it in `autoload-dev`